data/templates/css/wikihow.scss

```diff
@@ -22,39 +21,32 @@ $xga: 1024px * 36 / 56;
 $wxga: 1366px * 36 / 56;
 $fhd: 1920px * 36 / 56;
 
-$gray-darker: #191919;
-$gray-dark: #333;
-$gray: #666;
-$gray-light: #959595;
-$gray-lighter: #ccc;
+$text-color-body: #505050;
+$text-color-title: #282828;
 
-$fast: 0.25s;
-$normal: 0.5s;
-$slow: 1s;
+$section-color: #e9e9e3;
 
-//
-//
-//
+$divider-color: #ccc;
+$divider-color-composite: #959595;
+
+//////////////////////////////
+// Styling
+//////////////////////////////
 
 * {
     outline: none;
     -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
     &[align="center"] {
         text-align: center;
     }
 }
 
 html, body {
-    height: 100%;
+    font: 20px $font-primary;
 }
 
 body {
-    background: #ececec image-url("noise.png");
-}
-
-img {
-    vertical-align: middle;
+    background: #fff;
```

Comment:
Stupid consistency question, but in some files we go for #fff and in some files, white. Any particular reason?